labgrid/resource/udev.py

```diff
@@ -328,6 +328,17 @@ def filter_match(self, device):
 
         return super().filter_match(device)
 
+@target_factory.reg_resource
+@attr.s(eq=False)
+class TegraUSBLoader(USBResource):
+    def filter_match(self, device):
+        match = (device.properties.get('ID_VENDOR_ID'), device.properties.get('ID_MODEL_ID'))
+
+        if match not in [("0955", "7340")]:  # or 7740
+            return False
```

Comment:
If `7740` is another supported device, it should be added to the list to match against. See `class IMXUSBLoader` for an example of how to allow multiple VID/PID pairs.

Otherwise, the comment's meaning is not clear to me.

Reply:
I'll drop the comment. I just saw that number in some code but I don't have a device which uses it.